cmake/SpicyAnalyzer.cmake

```diff
@@ -0,0 +1,26 @@
+function(spicy_analyzer)
+    set(options)
+    set(oneValueArgs NAME KIND)
```

Comment:
I don't think we want people to have to specify a "kind". Seems the need for this is just an artifact of the current structure of `spicy-analyzers`. In a standalone analyzer one wouldn't really have that.

Reply:
This was purely an artifact of how we organized the source tree. I got rid of the extra `file` and `protocol` level under `analyzers`, so now we can do away with `KIND` and make this function available in general in zeek/spicy-plugin; I opened zeek/spicy-plugin#61 there.